vxsandbox/resources/http.py

```diff
@@ -237,7 +237,7 @@ def data_limit_check(response, length):
                     % (length, data_limit,))
             return response
 
-        if header is None:
+        if header is None or method.upper() in ['DELETE', 'HEAD']:
```

Comment:
This only needs to check for `HEAD`. `DELETE` shouldn't have a `Content-Length` header.